src/backend/api/geese-features/add.js

```diff
@@ -0,0 +1,16 @@
+import db from '../../db';
+
+export default (req, res) => {
+  const featureData = req.body;
+  db.geeseFeatures
+    .addFeature(featureData)
+    .then((response) => {
+      if (response) {
+        res.send(response);
+      }
```

Comment:
We should handle the response not existing as a 4xx client error here (say 403).